orchestra/utils/task_lifecycle.py

```diff
@@ -212,12 +212,22 @@ def get_task_assignment_details(task_assignment):
         TaskAssignment.objects.filter(
             task=task_assignment.task)
         .order_by('-assignment_counter')[0])
+
+    try:
+        worker = {
+            'first_name': task_assignment.worker.user.first_name,
+            'last_name': task_assignment.worker.user.last_name,
+        }
+    except AttributeError:
+        worker = None
```

Comment:
does it make sense to make `worker` have all of the keys it would above, but have them be None?  like checking for the actual source of the attribute error and then either getting the attributes or `None` per field?